docs/userguide/annotations.md

```diff
@@ -1616,11 +1616,8 @@ or
 
 Tags are defined as a comma separated list within the `--%tags` annotation. 
 
-When executing a test run with tag filter applied, the framework will find all tests associated with the given tags and execute them. 
-The framework applies `OR` logic to all specified tags so any test / suite that matches at least one tag will be included in the test run. 
-
-When a suite/context is tagged, all of its children will automatically inherit the tag and get executed along with the parent. Parent suite tests are not executed, but a suitepath hierarchy is kept.
-
+When a suite/context is tagged, all of its children will automatically inherit the tag and get executed along with the parent, unless they are excluded explicitly at runtime with a negated tag expression.
+Parent suite tests are not executed, but a suitepath hierarchy is kept.
```

Comment:
The sample code below is referenced in the example calls to `ut.run` that are now moved to `running-unit-tests.md`